### PR TITLE
docs: wrong schema in the static routing example config

### PR DESCRIPTION
**Pull request by ShiKaiWi**
# Which issue does this PR close?

Closes #

# Rationale for this change
 There are some bugs in the example config in the deployment doc.

# What changes are included in this PR?
Fix the wrong config in the deployment doc.

# Are there any user-facing changes?
None.

# How does this change test
No need.

### CODE REVIEW

File: docs/guides/src/deploy/static_routing.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ addr = '127.0.0.1'
 port = 8831
 
 [[static_route.topology.schema_shards]]
-schema = 'public_0'
```
Comment (jiacai2050) [inline docs/guides/src/deploy/static_routing.md:80]: Any difference between them?

Comment (ShiKaiWi) [reply docs/guides/src/deploy/static_routing.md:80]: The config content differs from the description in the static routing section.

Comment (jiacai2050) [reply docs/guides/src/deploy/static_routing.md:80]: Good catch

**Review by jiacai2050 — APPROVED** [review]
LGTM

**Pull request merged**